hedera-node/src/main/java/com/hedera/services/context/BasicTransactionContext.java

```diff
@@ -91,7 +93,8 @@ public class BasicTransactionContext implements TransactionContext {
 	private byte[] hash;
 	private boolean isPayerSigKnownActive;
 	private Instant consensusTime;
-	private TxnAccessor accessor;
+	private SignedTxnAccessor accessor;
+	private PlatformTxnAccessor platformTxnAccessor;
```

Comment:
Imo we don't need a separate field for the `PlatformTxnAccessor` type since it's just a refinement of `SignedTxnAccessor`. Also since we now have interfaces, we'd prefer to use them here instead of the implementation types. (Like how you declare `List<?>` instead of `ArrayList<?>`.) So just,
```suggestion
	private TxnAccessor accessor;
```

Reply:
Fixed. Used the interface instead.